src/state/shellscripts/bwrap.sh

```diff
@@ -72,28 +74,24 @@ done
 # that remain writeable. ccache seems widespread in some Fedora systems.
 add_ccache_mount() {
   if command -v ccache > /dev/null; then
-      ccache_dir_regex='cache_dir = (.*)$'
-      local IFS=$'\n'
-      for f in $(ccache -p 2>/dev/null); do
-        if [[ $f =~ $ccache_dir_regex ]]; then
-          ccache_dir=${BASH_REMATCH[1]}
-          break
-        fi
-      done
+      ccache_dir=$(ccache -p | awk -F '=' '/cache_dir/ { print $2; exit }')
       CCACHE_DIR=${CCACHE_DIR-$HOME/.ccache}
       ccache_dir=${ccache_dir-$CCACHE_DIR}
-      add_mounts rw "$ccache_dir"
+      add_mounts "rw" "$ccache_dir"
```

Comment:
```suggestion
      add_mounts rw "$ccache_dir"
```
Could you keep changes to the minimal?